README.md

```diff
@@ -1 +1,23 @@
-# Hello-World
\ No newline at end of file
+
+#Hello World
+This is the first exercise on the Microverse Week 1 Full Stack Development course. 
+
+##Built With
+HTML & CSS
+
```

Comment:
- [ ] Good job creating a readme file for your project 😄 
There are still some issues with it that need to be addressed 
  - Try using [Microverse readme template](https://github.com/microverseinc/readme-template) in your projects from now on
  - When writing headers in markup language make sure to leave a space between your `#` and the word. 
  - I highly recommend adding a license to your project: [Add license to repo](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can follow the steps in there and you will come up with something like this:
  
  
![Captura de Pantalla 2022-09-20 a la(s) 12 27 12 p  m](https://user-images.githubusercontent.com/86022690/191328701-925bbd0d-6550-4a12-a810-8656510d54ca.png)

If you have any questions regarding this issue, feel free to tag me here 😄

Reply:
@GiuseppeTG I used the Microverse Template for the README.md file on my second code review request. Where did I make a mistake?

Reply:
Please be more specific on where my README.md and Microverse's README.md differ.